Sources/PointFree/Database.swift

```diff
@@ -354,8 +373,14 @@ private func upsertUser(
 private func fetchUser(byEmail email: EmailAddress) -> EitherIO<Error, Database.User?> {
   return firstRow(
     """
-    SELECT "email", "github_user_id", "github_access_token", "id", "is_admin", "name", "subscription_id"
-    FROM "users"
```

Comment:
How's this still working without `FROM "users"`?

Reply:
jeez, good catch. it definitely wouldn't be. but also doesn't seem we actually use this endpoint, so gonna remove.